pallets/pooled-staking/src/calls.rs

```diff
@@ -113,9 +113,14 @@ impl<T: Config> Calls<T> {
     pub fn request_delegate(
         candidate: Candidate<T>,
         delegator: Delegator<T>,
-        pool: TargetPool,
+        pool: ActivePoolKind,
         stake: T::Balance,
     ) -> DispatchResultWithPostInfo {
+        ensure!(
+            !PausePoolsExtrinsics::<T>::get(),
+            Error::<T>::PoolsExtrinsicsArePaused
```

Comment:
I believe this storage item is not needed because ALL extrinsics for all pallets are paused during a multi block migration. But not sure, need to verify that

Reply:
Let's keep it just in case. It can also be useful at some point if we notice an issue with the pallet and what to reduce damages quickly.